app/src/main/AndroidManifest.xml

```diff
@@ -7,6 +7,10 @@
     <uses-permission android:name="android.permission.ACCESS_NETWORK_STATE" />
     <uses-permission android:name="android.permission.ACCESS_WIFI_STATE" />
 
+    <permission
+        android:name="tech.relaycorp.relaynet.GATEWAY_SYNC"
```

Comment:
Shouldn't this be called `tech.relaycorp.gateway.<whatever>` for consistency with the applicationId / package name?

Reply:
Could be, as it could also be global, but I can set it to `tech.relaycorp.relaynet.gateway.SYNC`

Reply:
Shouldn't `gateway` come before `relaynet` as in `tech.relaycorp.gateway.relaynet.SYNC`, since `tech.relaycorp.gateway` is the applicationId and package name?

Reply:
I don't know if there's a convention for this, but it does make more sense.